modules/apps/portal-search/portal-search-web/src/main/resources/resource-actions/default.xml

```diff
@@ -142,4 +142,24 @@
 			</guest-unsupported>
 		</permissions>
 	</portlet-resource>
+	<portlet-resource>
+		<portlet-name>com_liferay_portal_search_web_type_facet_portlet_TypeFacetPortlet</portlet-name>
```

Comment:
Alphabetize

Reply:
Ok! Pushed up a commit to alphabetize, and rebased atop a stable branch w/ cherry-picks.